docs/compatibility.md

```diff
@@ -23,9 +23,10 @@ versions of related services that each operator version is compatible with. The
 "Most Recent Version" column shows the last version of the operator that was
 published for each major version.
 
-| Operator Version  | Most Recent Version | Supported Cluster Models  | Supported FDB Versions  | Supported Kubernetes Versions |
-| ----------------- | ------------------- | ------------------------- | ----------------------- | ----------------------------- |
-| 0.x               | 0.51.0              | v1beta1                   | 6.1.12+                 | 1.15.0+                       |
+| Operator Version | Most Recent Version | Supported Cluster Models  | Supported FDB Versions | Supported Kubernetes Versions |
+|------------------|---------------------| ------------------------- |-----------------------|-------------------------------|
+| 0.* - 0.51.0     | 0.51.0              | v1beta1                   | 6.1.12+               | 1.15.0+                       |
+| 0.52.0+          | -                   | v1beta1                   | 6.2.0+                | 1.19.0+                       |
```

Comment:
I decided to increase the Kubernetes version to `1.19` since our tests are not testing against `1.15` so we might not support that version.